app/views/core/RootView.coffee

```diff
@@ -201,10 +201,11 @@ module.exports = class RootView extends CocoView
       #console.log 'Saved language:', newLang
 
   isOldBrowser: ->
-    if $.browser
+    if features.china and $.browser
```

Comment:
Presumably we still want to recommend getting a newer browser even when not on the China feature flag, although maybe not as aggressively? Don't think this this whole thing should go under feature flag.